src/backend/crossterm.rs

```diff
@@ -344,3 +370,300 @@ impl ModifierDiff {
         Ok(())
     }
 }
+
+impl From<CAttribute> for Modifier {
+    fn from(value: CAttribute) -> Self {
+        Modifier::from(CAttributes::from(value))
```

Comment:
Can you add a doc comment for this line - it's a bit weird / requires understanding of crossterm Attribute vs Attributes

Reply:
Sure, plus the difference is subtle